slices/slice_func_test.go

```diff
@@ -164,6 +165,25 @@ func TestMap(t *testing.T) {
 	f([]int{1, 2, 3}, []int{2, 4, 6})
 }
 
+func TestMapFilter(t *testing.T) {
```

Comment:
Could you also add an `Example`, please? They are in `examples_test.go`. I think examples are very helpful for the end users.